ingestion/src/metadata/ingestion/source/pipeline/dagster/connection.py

```diff
@@ -32,14 +32,15 @@ def get_connection(connection: DagsterConnection) -> DagsterGraphQLClient:
     """
     Create connection
     """
-    url = connection.host
+    url = connection.host[:-1] if connection.host.endswith("/") else connection.host
```

Comment:
```suggestion
    url = connection.host
```

you can drop this change as I'll be handling that with my pr where I've created a common function for this

Reply:
Ok 👍